src/Codeception/Template/Upgrade4.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Codeception\Template;
+
+use Codeception\Configuration;
+use Codeception\InitTemplate;
+
+class Upgrade4 extends InitTemplate
+{
+    const SURVEY_LINK = 'http://bit.ly/codecept-survey';
+    const DONATE_LINK = 'https://opencollective.com/codeception';
+
+    public function setup()
+    {
+        if (!$this->isInstalled()) {
+            $this->sayWarning('Codeception is not installed in this dir.');
+            return;
+        }
+        $this->sayInfo('Welcome to Codeception v4 Upgrade wizard!');
+        $this->say('');
+        $this->say('Codeception is maintained since 2011, is free & open-source.');
+        $this->say('To make it better we need your feedback on it!');
+        $this->say('');
+        $this->say('Please take a minute and fill in a brief survey:');
+        $this->say('<bold>'  . self::SURVEY_LINK . '</bold>');
+        sleep(5);
+        $this->say('');
+        $result = $this->ask('<question>Did you fill in the survey?</question>', true);
+        if ($result) {
+            $this->say('Thank you! ');
+        } else {
+            $this->say('Anyway...');
+        }
+        $config = Configuration::config();
+        $modules = [];
+        $suites = Configuration::suites();
+        if (empty($suites)) {
+            $this->sayError("No suites found in current config.");
+            $this->sayWarning('If you use sub-configs with `include` option, run this script on subconfigs:');
+            $this->sayWarning('Example: php /vendor/bin/codecept init upgrade4 -c backend/');
```

Comment:
Please change `Example: php /vendor/bin/codecept`
to `Example: php vendor/bin/codecept`

Reply:
Ouch, right!

Reply:
Btw there is a way you could add suggestions so I can commit them automatically 
 
![Selection_664](https://user-images.githubusercontent.com/220264/70322378-a5f5c400-1832-11ea-82aa-6a4572de00b9.png)

Reply:
```suggestion
            $this->sayWarning('Example: php vendor/bin/codecept init upgrade4 -c backend/');
```